cmds/boot/systemboot/systemboot_linux.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"os"
+	"syscall"
+)
+
+const (
+	_IOC_NONE = 0x0
+
+	_IOC_NRBITS   = 8
+	_IOC_TYPEBITS = 8
+	_IOC_SIZEBITS = 14
+	_IOC_NRSHIFT  = 0
+
+	_IOC_TYPESHIFT = _IOC_NRSHIFT + _IOC_NRBITS
+	_IOC_SIZESHIFT = _IOC_TYPESHIFT + _IOC_TYPEBITS
+	_IOC_DIRSHIFT  = _IOC_SIZESHIFT + _IOC_SIZEBITS
+
+	_IOC_NVRAM = 'p'
+)
+
+func ioc(dir int, t int, nr int, size int) int {
```

Comment:
Most of this stuff is already in:
https://github.com/u-root/u-root/blob/master/pkg/ipmi/ioctl_unix.go

I would avoid duplicating it. The options are:
* Export the functions in the ipmi package so that they are usable elsewehere
* Create an ioctl package

Reply:
I exported the functions in the ipmi package.